File: module_wanandroid/src/main/java/com/yyxnb/module_wanandroid/bean/WanAriticleBean.java
package com.yyxnb.module_wanandroid.bean;

import java.util.List;

public class WanAriticleBean {


    /**
     * apkLink :
     * audit : 1
     * author :
     * canEdit : false
     * chapterId : 502
     * chapterName : 自助
     * collect : false
     * courseId : 13
     * desc :
     * descMd :
     * envelopePic :
     * fresh : false
     * id : 14091
     * link : https://juejin.im/post/5ef9327e6fb9a07e716acac4
     * niceDate : 2020-06-30 08:24
     * niceShareDate : 2020-06-30 08:24
     * origin :
     * prefix :
     * projectLink :
     * publishTime : 1593476689000
     * realSuperChapterId : 493
     * selfVisible : 0
     * shareDate : 1593476689000
     * shareUser : JsonChao
     * superChapterId : 494
     * superChapterName : 广场Tab
     * tags : []
     * title : 深入探索 Gradle 自动化构建技术（九、Gradle 插件平台化框架 ByteX 探秘之旅）
     * type : 0
     * userId : 611
     * visible : 1
     * zan : 0
     */

    public String apkLink;
    public int audit;
    public String author;
    public boolean canEdit;
    public int chapterId;
    public String chapterName;
    public boolean collect;
    public int courseId;
    public String desc;
    public String descMd;
    public String envelopePic;
    public boolean fresh;
    public int id;
    public String link;
    public String niceDate;
    public String niceShareDate;
    public String origin;
    public String prefix;
    public String projectLink;
    public long publishTime;
    public int realSuperChapterId;
    public int selfVisible;
    public long shareDate;
    public String shareUser;
    public int superChapterId;
    public String superChapterName;
    public String title;
    public int type;
    public int userId;
    public int visible;
    public int zan;
    public List<DataBeans> tags;

    // 轮播的
    public String imagePath;
    public String url;
    public int isVisible;
    public int order;

    public static class DataBeans{
        public String name;
        public String url;
    }

}

File: module_wanandroid/src/main/java/com/yyxnb/module_wanandroid/bean/WanAriticleBean.kt
package com.yyxnb.module_wanandroid.bean

data class WanAriticleBean(
        /**
         * apkLink :
         * audit : 1
         * author :
         * canEdit : false
         * chapterId : 502
         * chapterName : 自助
         * collect : false
         * courseId : 13
         * desc :
         * descMd :
         * envelopePic :
         * fresh : false
         * id : 14091
         * link : https://juejin.im/post/5ef9327e6fb9a07e716acac4
         * niceDate : 2020-06-30 08:24
         * niceShareDate : 2020-06-30 08:24
         * origin :
         * prefix :
         * projectLink :
         * publishTime : 1593476689000
         * realSuperChapterId : 493
         * selfVisible : 0
         * shareDate : 1593476689000
         * shareUser : JsonChao
         * superChapterId : 494
         * superChapterName : 广场Tab
         * tags : []
         * title : 深入探索 Gradle 自动化构建技术（九、Gradle 插件平台化框架 ByteX 探秘之旅）
         * type : 0
         * userId : 611
         * visible : 1
         * zan : 0
         */
        var apkLink: String? = null,
        var audit: Int = 0,
        @JvmField
        var author: String? = null,
        var canEdit: Boolean = false,
        var chapterId: Int = 0,
        @JvmField
        var chapterName: String? = null,
        var collect: Boolean = false,
        var courseId: Int = 0,
        var desc: String? = null,
        var descMd: String? = null,
        @JvmField
        var envelopePic: String? = null,
        @JvmField
        var fresh: Boolean = false,
        var id: Int = 0,
        var link: String? = null,
        @JvmField
        var niceDate: String? = null,
        var niceShareDate: String? = null,
        var origin: String? = null,
        var prefix: String? = null,
        var projectLink: String? = null,
        var publishTime: Long = 0,
        var realSuperChapterId: Int = 0,
        var selfVisible: Int = 0,
        var shareDate: Long = 0,
        @JvmField
        var shareUser: String? = null,
        var superChapterId: Int = 0,
        @JvmField
        var superChapterName: String? = null,
        @JvmField
        var title: String? = null,
        @JvmField
        var type: Int = 0,
        var userId: Int = 0,
        var visible: Int = 0,
        var zan: Int = 0,
        var tags: List<DataBeans>? = null,

        // 轮播的
        var imagePath: String? = null,
        var url: String? = null,
//    var isVisible: Int = 0
        var order: Int = 0


)

data class DataBeans(
        var name: String? = null,
        var url: String? = null
)